integrations/osmosis/packages/abstract-adapter/src/lib.rs

```diff
@@ -1,4 +1,4 @@
 pub const OSMOSIS: &str = "osmosis";
-pub const AVAILABLE_CHAINS: &[&str] = &["osmosis"];
+pub const AVAILABLE_CHAINS: &[&str] = &["osmosis", "osmo-test"];
```

Comment:
:rocket:

Reply:
And we should probably use `abstract_sdk::core::registry::OSMOSIS` here, as long as we include `osmo-test` there.